docs/custom-metric-filter.md

```diff
@@ -0,0 +1,146 @@
+Monarch (Google's internal metrics aggregator).
+
+## Background
+
+When Config Sync is installed on a GKE cluster where a
+[default service account](http://cloud/kubernetes-engine/docs/tutorials/authenticating-to-cloud-platform)
+is available, or on a GKE cluster that has
+[Workload Identity](http://cloud/kubernetes-engine/docs/how-to/workload-identity)
+enabled and
+[IAM is correctly setup](https://cloud.google.com/anthos-config-management/docs/how-to/monitoring-config-sync#custom-monitoring),
+Config Sync will try to apply the Open Telemetry Collector config map that contains
+[googlecloud exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/googlecloudexporter)
+to write metrics into Cloud Monitoring. The default configuration contains a
+bare minimum list of metrics, but some customers might want to configure
+different backends or fine tune the metric list for better monitoring solutions.
+
+This guide includes instructions for how to adjust filters, modify exporters, 
+or opt out of exporting metrics all together.
+
+Note: This workaround applies to Config Sync v1.8+, when custom monitoring
+was integrated.
+
+## Default metrics pipeline config (v1.12+)
+
+Config Sync will check for the default credential in the environment when
+starting up. For a GKE cluster which always have a default service account,
+Config Sync will deploy the following Open Telemetry Collector configuration to
+enable the Google Cloud monitoring pipelines.
+
+A pipeline consist of:
+
+-   receivers: receives metrics from all reconcilers inside Config Sync
+-   processors: filters or transforms metrics before sending out
+-   exporters: various backend endpoints and their configurations
+
+Config Sync uses
+[filter processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/filterprocessor)
+to control the exclusive list of metrics that goes into each monitoring backend.
+
+Config Sync uses two major exporters: [googlecloud exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/googlecloudexporter)
+[prometheus exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/prometheusexporter)
+
+- `googlecloud` stands for the configuration that exports to the Cloud
+Monitoring backend.
+- `googlecloud/kubernetes` stands for the configuration that exports to the
+Cloud Monarch backend
+- `prometheus` stands for the configuration that exports metrics in Prometheus
+format and later can be scraped by a prometheus server following [this guide](http://cloud/anthos-config-management/docs/how-to/monitoring-config-sync#prometheus)
+
+Here is [the current configuration](https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/pkg/metrics/otel.go#L38) of Open Telemetry Collector
+in Config Sync.
+
+## Patch the otel collector deployment with Configmap
+
+1. **Get the config map**
+
+```
```

Comment:
this section needs 4 space indentation to be under the numbered bullet

Reply:
Done